app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :created_at, :name, :email, :password
+  has_many :lists
+  has_many :items
+
+  # Delegate the practical definition of `full_name` to
```

Comment:
yeah same here and the methods here should be in the models

for example...
class User < ApplicationRecord
  has_many :lists, dependent: :destroy
  validates :name, presence: true
  validates :email, presence: true
  validates :password_digest, presence: true
end